velox/exec/OperatorTraceWriter.h

```diff
@@ -70,4 +71,29 @@ class OperatorTraceWriter {
   bool finished_{false};
 };
 
+/// Used to write the input splits during the execution of a traced 'TableScan'
+/// operator.
+///
+/// Currently, it only works with 'HiveConnectorSplit'. In the future, it will
+/// be extended to handle more types of splits, such as
+/// 'IcebergHiveConnectorSplit'.
+class OperatorTraceSplitWriter {
```

Comment:
I think it might be good to write summary as well for table scan so we can know if a particular table scan operator uses a lot of memory? Thanks!